data/json/speech.json

```
@@ -423,13 +423,13 @@
     "type": "speech",
     "speaker": [ "mon_mi_go", "mon_mi_go_slaver" ],
     "sound": "\"Do you understand what I'm saying?\"",
-    "volume": 30
+    "volume": 20
   },
   {
     "type": "speech",
     "speaker": [ "mon_mi_go", "mon_mi_go_slaver", "mon_mi-go-surgeon" ],
     "sound": "\"Look, it's responding!\"",
-    "volume": 40
+    "volume": 32
```

```suggestion
    "volume": 26
```
This one is not likely to have been done at full "scream" volume.